main/src/main/java/com/sedmelluq/discord/lavaplayer/source/soundcloud/SoundCloudAudioTrack.java

```diff
@@ -64,7 +64,9 @@ private void playFromIdentifier(
         if (!recursion) {
             // Old "track ID" entry? Let's "load" it to get url.
             AudioTrack track = sourceManager.loadTrack(trackInfo.uri);
-            playFromIdentifier(httpInterface, track.getIdentifier(), true, localExecutor);
+            if (track != null) {
```

Comment:
This should throw an exception to say that the track could not be played/loaded in instances where null is returned, to prevent confusing behaviour where a track appears to play but actually doesn't, silently

Reply:
Upon second glance I have decided to ignore the filter in this case, and have removed the null check